api/src/test/java/uk/gov/hmcts/fees2/register/api/controllers/FeeControllerTest.java

```diff
@@ -272,6 +269,99 @@ public void findFeeWithVolume_inWholeNumber_shouldReturnValidFee() throws Except
         forceDeleteFee(arr[3]);
     }
 
+    @Test
+    @Transactional
+    public void findFeeLookupWithKeyword_whenFeeHasKeyword() throws Exception {
+        FixedFeeDto fixedFeeDto = FeeDataUtils.getCreateProbateCopiesFeeRequest();
+        fixedFeeDto.setKeyword("testKeyword");
+        String loc = saveFeeAndCheckStatusIsCreated(fixedFeeDto);
+
+        String[] arr = loc.split("/");
+
+        MvcResult result = restActions
+            .withUser("admin")
+            .get("/fees-register/fees/lookup?service=probate&jurisdiction1=family&jurisdiction2=probate registry&channel=default&event=copies&applicant_type=all&amount_or_volume=3&keyword=testKeyword")
+            .andExpect(status().isOk())
+            .andReturn();
+
+        forceDeleteFee(arr[3]);
+    }
+
+    @Test
+    @Transactional
+    public void findFeeLookupWithWrongKeyword_whenFeeHasKeyword() throws Exception {
+        FixedFeeDto fixedFeeDto = FeeDataUtils.getCreateProbateCopiesFeeRequest();
+        fixedFeeDto.setKeyword("testKeyword");
+        String loc = saveFeeAndCheckStatusIsCreated(fixedFeeDto);
+
+        String[] arr = loc.split("/");
+
+        MvcResult result = restActions
+            .withUser("admin")
+            .get("/fees-register/fees/lookup?service=probate&jurisdiction1=family&jurisdiction2=probate registry&channel=default&event=copies&applicant_type=all&amount_or_volume=3&keyword=wrongKey")
+            .andExpect(status().isNotFound())
+            .andReturn();
+
+        forceDeleteFee(arr[3]);
```

Comment:
if there is an exception then it will leave the test data behind. so pls move to tear down